### DISCUSSION TITLE
2024/11/15/%E7%BD%91%E5%AE%89%E5%AE%9E%E8%B7%B55-2/

### DISCUSSION THREAD

**Discussion by giscus** [moyigeek/blog#4] [Announcements]
# 2024/11/15/%E7%BD%91%E5%AE%89%E5%AE%9E%E8%B7%B55-2/

IP 地址国别信息查询系统设计本实验旨在探讨如何使用xdb文件格式（本项目中设计的一种高效且可自定义的数据格式，用于存储和查询 IP 地址相关信息）在海量数据集下实现高效的ip 国别信息查询。实验重点包括理解xdb的数据结构与查询流程，结合 IPv4和IPv6 的实现代码进行分析。同时，实验将使用pyshark捕获实时网络流量并查询IP 的国别信息，进一步验证查询的实用性和性能。 完成过程 理解x

https://blog.moyihust.eu.org/2024/11/15/%E7%BD%91%E5%AE%89%E5%AE%9E%E8%B7%B55-2/

**Comment by moyigeek** [comment]
wow
